src/topp/FileMerger.cpp

```diff
@@ -123,7 +124,10 @@ class TOPPFileMerger :
     setValidFormats_("out", ListUtils::create<String>("mzML,featureXML,consensusXML,traML,fasta"));
 
     registerFlag_("annotate_file_origin", "Store the original filename in each feature using meta value \"file_origin\" (for featureXML and consensusXML only).");
-    registerStringOption_("append_method", "<choice>", "append_rows", "Append consensusMaps rowise or colwise. (Please use colwise for the MSstatsConverter)", false);
+    registerStringOption_("append_method", "<choice>", "append_rows", "(ConsensusXML-only) Append quantitative information about features row-wise or column-wise.\n"
```

Comment:
the concept of rows and columns it a bit confusing here, because a user has probably no relation to it.
append_rows basically means treat the underlying featuremaps independently and simply append them?
Whereas append_rows basically means: merge the i
th featuremap from all maps.
alternatively: Maybe its enough to say that columns are featuremaps, and rows are input consensusmaps?

Reply:
Hmm not sure if that is correct. I think both are appending to make one consensusMap. But row-wise will keep the map_index numbers (e.g. 1,2,3 and 1,2,3) and use the file information from the first consensusMap for the new extended map.  Col-wise will update the map_index numbers to (1,2,3,4,5,6) and add the corresponding file information to the ConsensusXML header.

Reply:
@timosachsenberg implemented that mainly and might be able to comment more.